tools/accuracy/accuracy_checker.py

```diff
@@ -125,28 +168,80 @@ def main():
     test_inputs = {}
     for name, shape in model.get_parameter_shapes().items():
         if args.verbose:
-            print('Parameter {} -> {}'.format(name, shape))
+            print(f'Parameter {name} -> {shape}')
         in_shape = shape.lens()
         in_type = shape.type_string()
-        if not args.fill1:
+        if not args.fill1 and not args.fill0:
             test_input = np.random.rand(*(in_shape)).astype(
                 get_np_datatype(in_type))
-        else:
+        elif not args.fill0:
             test_input = np.ones(in_shape).astype(get_np_datatype(in_type))
+        else:
+            test_input = np.zeros(in_shape).astype(get_np_datatype(in_type))
         test_inputs[name] = test_input
         params[name] = migraphx.to_gpu(migraphx.argument(test_input))
 
     pred_migx = np.array(migraphx.from_gpu(model.run(params)[-1]))
 
-    sess = ort.InferenceSession(model_name, providers=[args.provider])
+    if use_onnx:
+        sess = ort.InferenceSession(model_name, providers=[args.provider])
+
+        ort_params = {}
+        for input in sess.get_inputs():
+            ort_params[input.name] = test_inputs[input.name]
+
+        try:
+            pred_fw = sess.run(None, ort_params)[-1]
+        except Exception as e:
+            if any(input_dims):
+                print(
+                    'Error: custom input dim may not be compatible with onnx runtime'
+                )
+            raise e
+    else:
+        import tensorflow as tf
+
+        def load_tf_graph(model_name):
+            with tf.io.gfile.GFile(model_name, 'rb') as f:
+                graph_def = tf.compat.v1.GraphDef()
+                graph_def.ParseFromString(f.read())
+
+            with tf.compat.v1.Graph().as_default() as graph:
+                tf.graph_util.import_graph_def(graph_def)
+            return graph
+
+        graph = load_tf_graph(model_name)
+        is_nhwc = False
+        graph_ops = []
+        for op in graph.get_operations():
+            graph_ops.append(op.name)
+            if 'Conv' in op.node_def.op and not args.nchw:
+                is_nhwc = True
```

Comment:
https://www.tensorflow.org/api_docs/python/tf/Operation#get_attr 

you can use `get_attr('data_format")`